coordinate.go

```diff
@@ -92,7 +93,7 @@ func (a *Agent) runNodePing(node *api.Node) {
 	key := fmt.Sprintf("%sprobes/%s", a.config.KVPath, node.Node)
 	kvPair, _, err := kvClient.Get(key, nil)
 	if err != nil {
-		a.logger.Printf("[ERR] could not get critical status for node %q: %v", node.Node, err)
+		a.logger.Error("could not get critical status for node", "info", hclog.Fmt("%q: %v", node.Node, err))
```

Comment:
Can you split "info" k/v pair into something like `"node", node.Node, "error", err`?